web/packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -369,6 +369,13 @@ export default class MainProcess {
 
   private _setAppMenu() {
     const isMac = this.settings.platform === 'darwin';
+    const commonHelpTemplate: MenuItemConstructorOptions[] = [
+      { label: 'Learn More', click: openDocsUrl },
```

Comment:
While we're at it, what if we just changed it to "Open Documentation"? It's more direct than "Learn More".